obelisk/cpp/obelisk_cpp/include/obelisk_controller.h

```diff
@@ -0,0 +1,103 @@
+#pragma once
+
+#include "obelisk_node.h"
+
+namespace obelisk {
+
+    template <typename ControlMessageT, typename EstimatorMessageT> class ObeliskController : public ObeliskNode {
+      public:
+        explicit ObeliskController(const std::string& name, const std::string& ctrl_key = "pub_ctrl",
+                                   const std::string& est_key = "sub_est", const std::string& timer_key = "timer_ctrl")
```

Comment:
Having thought about it some more, since they know type of component they are getting (i.e. by accessing the proper dict, or by using the proper access function), we actually don't need to annotate the type of component in the name of the key. So "sub_est" -> "est" and four you "subscriber_est" -> "est". Let me know what you think about this.

Reply:
Decided to just match the python version for now, but I added a comment on your PR about making local variables to hold these values.